test/helpers/utils.go

```diff
@@ -380,6 +380,40 @@ func WaitForNodeConfigChange(t *testing.T, cs *framework.ClientSet, node corev1.
 	return nil
 }
 
+func WaitForNodeReady(t *testing.T, cs *framework.ClientSet, node corev1.Node) error {
```

Comment:
thought (non-blocking): I vaguely recall there being a helper function very similar to this, but I'm not 100% sure.

Reply:
I thought so, too, but I couldn't find one looking through the helpers. I did end up adding the `WaitForNodeReady` helper but I may have overlooked it. I just parsed through the helpers again and didn't see anything that would be a great fit (unless I overlooked it again)